### PR TITLE
Update scraper README to reference main README for project setup.

### PR DESCRIPTION
**Pull request by markfickett**
They disagree slightly, and the top-level README has more complete/reliable instructions.

I also added a note about python default versions for Ubuntu.

### CODE REVIEW

File: recidiviz/tools/scraper_template/README.md [hunk 1]
```diff
@@ -1,9 +1,6 @@
 Creating a Scraper
 ==================
-Before doing anything, setup your environment by running `pipenv install --dev` in the top level directory.
-Enter your environment with `pipenv shell`. Finally, test that your system is all setup for scraper
-development by running an existing scraper using, say
-`python -m recidiviz.tools.run_scraper --region us_pa_greene`.
+Before doing anything, setup your environment according to the top-level README.
```
Comment (colincadams) [inline recidiviz/tools/scraper_template/README.md]: This should be made a link to that README

**Comment by markfickett** [comment]
Done & pushed. Do you squash and merge? If not I can squash and re-push.

File: recidiviz/tools/scraper_template/README.md (hunk 1 above)
Comment (markfickett) [reply recidiviz/tools/scraper_template/README.md]: Done.

**Comment by jamwalla** [comment]
Yep, we'll merge this in, sorry to leave it hanging! 🙏🏽🙏🏽

**Pull request merged**